libs/test-utils/Cargo.toml

```diff
@@ -15,14 +15,14 @@ cfg-primitives = { path = "../primitives", default-features = false }
 cfg-traits = { path = "../traits", default-features = false }
 cfg-types = { path = "../types", default-features = false }
 
```

Comment:
According to my search, this pallet has 1 dependency which does not make use of workspace inheritance.